processing file asmFiles/count_days.asm (1 of 30)
processing file asmFiles/fib.asm (2 of 30)
--- memsim.hex	2024-09-25 10:01:26.294915654 -0400
+++ memcpu.hex	2024-09-25 10:01:26.275914933 -0400
@@ -14,27 +14,16 @@
 :04000D00FE0814E3F2
 :04000E00FFFFFFFFF2
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400230000000001D8
+:0400250000000001D6
+:0400270000000001D4
+:0400290000000001D2
+:04002B0000000001D0
+:04002D0000000001CE
+:04002F0000000001CC
+:0400310000000001CA
+:0400330000000001C8
+:0400350000000001C6
+:0400370000000001C4
 :0403C0000000001623
 :00000001FF
processing file asmFiles/mergesort.asm (3 of 30)
--- memsim.hex	2024-09-25 10:01:28.639004552 -0400
+++ memcpu.hex	2024-09-25 10:01:28.614003604 -0400
@@ -86,60 +86,60 @@
 :0400550000078463B9
 :04005600FE9FF06FAA
 :0400570000008067BE
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400C0000000000339
+:0400C1000000000734
+:0400C2000000000733
+:0400C3000000000732
+:0400C4000000000830
+:0400C500000000082F
+:0400C600000000082E
+:0400C700000000092C
+:0400C8000000000A2A
+:0400C9000000000F24
+:0400CA000000001022
+:0400CB000000001120
+:0400CC000000001719
+:0400CD000000001A15
+:0400CE00000000200E
+:0400CF00000000200D
+:0400D0000000002309
+:0400D1000000002407
+:0400D2000000002604
+:0400D3000000002603
+:0400D4000000002800
+:0400D5000000002FF8
+:0400D6000000005ACC
+:0400D7000000005ACB
+:0400D8000000005ACA
+:0400D9000000005AC9
+:0400DA000000005AC8
+:0400DB000000005AC7
+:0400DC000000005AC6
+:0400DD000000005AC5
 :0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
+:0400DF000000005AC3
+:0400E0000000005AC2
+:0400E10000000030EB
+:0400E20000000030EA
+:0400E30000000030E9
+:0400E40000000030E8
+:0400E50000000030E7
+:0400E60000000030E6
+:0400E70000000030E5
+:0400E80000000030E4
+:0400E90000000030E3
+:0400EA0000000030E2
+:0400EB0000000030E1
+:0400EC0000000030E0
+:0400ED0000000030DF
+:0400EE0000000030DE
+:0400EF0000000030DD
+:0400F00000000030DC
+:0400F10000000030DB
+:0400F20000000030DA
+:0400F30000000030D9
+:0400F40000000030D8
+:0400F50000000030D7
 :0400F60000000030D6
 :0400F70000000032D3
 :0400F80000000041C3
@@ -151,69 +151,38 @@
 :0400FE000000005CA2
 :0400FF000000005CA1
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0401400000000002B9
+:0401410000000007B3
+:0401420000000002B7
+:040143000000003088
+:040144000000003087
+:040145000000003086
+:040146000000003085
+:040147000000003084
+:040148000000003083
+:040149000000003082
+:04014A000000003081
+:04014B000000003080
+:04014C00000000307F
+:04014D00000000307E
+:04014E00000000307D
+:04014F00000000307C
+:04015000000000307B
+:04015100000000307A
+:040152000000003079
+:040153000000003078
+:040154000000003077
+:040155000000003076
+:040156000000003075
+:040157000000003272
+:040158000000004162
+:040159000000004260
+:04015A00000000435E
+:04015B00000000445C
+:04015C000000005946
+:04015D000000005B43
+:04015E000000005C41
+:04015F000000005C40
+:040160000000006437
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (4 of 30)
processing file asmFiles/mult_procedure.asm (5 of 30)
processing file asmFiles/search.asm (6 of 30)
processing file asmFiles/test.beq_ntaken.asm (7 of 30)
processing file asmFiles/test.beq_taken.asm (8 of 30)
processing file asmFiles/test.bge_ntaken.asm (9 of 30)
processing file asmFiles/test.bge_taken.asm (10 of 30)
processing file asmFiles/test.bgeu_ntaken.asm (11 of 30)
processing file asmFiles/test.bgeu_taken.asm (12 of 30)
processing file asmFiles/test.blt_ntaken.asm (13 of 30)
processing file asmFiles/test.blt_taken.asm (14 of 30)
processing file asmFiles/test.bltu_ntaken.asm (15 of 30)
processing file asmFiles/test.bltu_taken.asm (16 of 30)
processing file asmFiles/test.bne_ntaken.asm (17 of 30)
processing file asmFiles/test.bne_taken.asm (18 of 30)
processing file asmFiles/test.branchjump1.asm (19 of 30)
processing file asmFiles/test.cfhazard_beq.asm (20 of 30)
processing file asmFiles/test.dhazard_add.asm (21 of 30)
processing file asmFiles/test.dhazard_lw.asm (22 of 30)
processing file asmFiles/test.halt.asm (23 of 30)
processing file asmFiles/test.jal.asm (24 of 30)
processing file asmFiles/test.jalr.asm (25 of 30)
processing file asmFiles/test.ldst1.asm (26 of 30)
processing file asmFiles/test.ldst2.asm (27 of 30)
processing file asmFiles/test.loadstore.asm (28 of 30)
processing file asmFiles/test.rtype.asm (29 of 30)
processing file asmFiles/test.shazard_lw.asm (30 of 30)
processing file asmFiles/count_days.asm (1 of 30)
--- memsim.hex	2024-09-25 10:02:37.998634974 -0400
+++ memcpu.hex	2024-09-25 10:02:37.977634177 -0400
@@ -1,4 +1,4 @@
-:04000000FFC0011329
+:0400000000002238A2
 :0400010001B00293B5
 :040002000080031364
 :040003007E80039365
@@ -36,6 +36,7 @@
 :04002300FFC1011305
 :0400240000A12023F4
 :0400250000008067F0
-:043FFD0000000018A8
-:043FFE000000223865
+:043FFD0000000007B9
+:043FFE0000000018A7
+:043FFF000000016D50
 :00000001FF
processing file asmFiles/fib.asm (2 of 30)
processing file asmFiles/mergesort.asm (3 of 30)
--- memsim.hex	2024-09-25 10:02:41.779778371 -0400
+++ memcpu.hex	2024-09-25 10:02:41.757777537 -0400
@@ -86,60 +86,60 @@
 :0400550000078463B9
 :04005600FE9FF06FAA
 :0400570000008067BE
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400C0000000000339
+:0400C1000000000734
+:0400C2000000000733
+:0400C3000000000732
+:0400C4000000000830
+:0400C500000000082F
+:0400C600000000082E
+:0400C700000000092C
+:0400C8000000000A2A
+:0400C9000000000F24
+:0400CA000000001022
+:0400CB000000001120
+:0400CC000000001719
+:0400CD000000001A15
+:0400CE00000000200E
+:0400CF00000000200D
+:0400D0000000002309
+:0400D1000000002407
+:0400D2000000002604
+:0400D3000000002603
+:0400D4000000002800
+:0400D5000000002FF8
+:0400D6000000005ACC
+:0400D7000000005ACB
+:0400D8000000005ACA
+:0400D9000000005AC9
+:0400DA000000005AC8
+:0400DB000000005AC7
+:0400DC000000005AC6
+:0400DD000000005AC5
 :0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
+:0400DF000000005AC3
+:0400E0000000005AC2
+:0400E10000000030EB
+:0400E20000000030EA
+:0400E30000000030E9
+:0400E40000000030E8
+:0400E50000000030E7
+:0400E60000000030E6
+:0400E70000000030E5
+:0400E80000000030E4
+:0400E90000000030E3
+:0400EA0000000030E2
+:0400EB0000000030E1
+:0400EC0000000030E0
+:0400ED0000000030DF
+:0400EE0000000030DE
+:0400EF0000000030DD
+:0400F00000000030DC
+:0400F10000000030DB
+:0400F20000000030DA
+:0400F30000000030D9
+:0400F40000000030D8
+:0400F50000000030D7
 :0400F60000000030D6
 :0400F70000000032D3
 :0400F80000000041C3
@@ -151,69 +151,59 @@
 :0400FE000000005CA2
 :0400FF000000005CA1
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
+:0401400000000007B4
+:0401410000000007B3
+:0401420000000007B2
+:0401430000000008B0
+:0401440000000008AF
+:0401450000000008AE
+:0401460000000009AC
+:040147000000000AAA
+:040148000000000FA4
+:0401490000000010A2
+:04014A0000000011A0
+:04014B000000001799
+:04014C000000001A95
+:04014D00000000208E
+:04014E00000000208D
+:04014F000000002389
+:040150000000002487
+:040151000000002684
+:040152000000002683
+:040153000000002880
+:040154000000002F78
+:040155000000003076
+:040156000000005A4B
+:040157000000003074
+:040158000000003073
+:040159000000003072
+:04015A000000003071
+:04015B000000003070
+:04015C00000000306F
+:04015D00000000306E
+:04015E00000000306D
+:04015F00000000306C
+:04016000000000306B
+:04016100000000306A
+:040162000000003069
+:040163000000003068
 :040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
+:040165000000003066
+:040166000000003065
+:040167000000003064
+:040168000000003063
+:040169000000003062
+:04016A000000003061
+:04016B000000003060
+:04016C00000000325D
+:04016D00000000414D
 :04016E00000000424B
 :04016F000000004349
 :040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:040171000000005931
+:040172000000005B2E
+:040173000000005C2C
+:040174000000005C2B
+:040175000000006422
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (4 of 30)
--- memsim.hex	2024-09-25 10:02:43.176831353 -0400
+++ memcpu.hex	2024-09-25 10:02:43.160830747 -0400
@@ -20,5 +20,6 @@
 :0400130000A1202305
 :04001400FFFFFFFFEC
 :043FFD0000000005BB
-:043FFE0000000014AB
+:043FFE0000000004BB
+:043FFF0000000014AA
 :00000001FF
processing file asmFiles/mult_procedure.asm (5 of 30)
processing file asmFiles/count_days.asm (1 of 30)
processing file asmFiles/fib.asm (2 of 30)
--- memsim.hex	2024-09-25 10:05:26.938041897 -0400
+++ memcpu.hex	2024-09-25 10:05:26.918041139 -0400
@@ -14,27 +14,16 @@
 :04000D00FE0814E3F2
 :04000E00FFFFFFFFF2
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400230000000001D8
+:0400250000000001D6
+:0400270000000001D4
+:0400290000000001D2
+:04002B0000000001D0
+:04002D0000000001CE
+:04002F0000000001CC
+:0400310000000001CA
+:0400330000000001C8
+:0400350000000001C6
+:0400370000000001C4
 :0403C0000000001623
 :00000001FF
processing file asmFiles/mergesort.asm (3 of 30)
--- memsim.hex	2024-09-25 10:05:29.285130909 -0400
+++ memcpu.hex	2024-09-25 10:05:29.257129847 -0400
@@ -86,60 +86,60 @@
 :0400550000078463B9
 :04005600FE9FF06FAA
 :0400570000008067BE
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400C0000000000339
+:0400C1000000000734
+:0400C2000000000733
+:0400C3000000000732
+:0400C4000000000830
+:0400C500000000082F
+:0400C600000000082E
+:0400C700000000092C
+:0400C8000000000A2A
+:0400C9000000000F24
+:0400CA000000001022
+:0400CB000000001120
+:0400CC000000001719
+:0400CD000000001A15
+:0400CE00000000200E
+:0400CF00000000200D
+:0400D0000000002309
+:0400D1000000002407
+:0400D2000000002604
+:0400D3000000002603
+:0400D4000000002800
+:0400D5000000002FF8
+:0400D6000000005ACC
+:0400D7000000005ACB
+:0400D8000000005ACA
+:0400D9000000005AC9
+:0400DA000000005AC8
+:0400DB000000005AC7
+:0400DC000000005AC6
+:0400DD000000005AC5
 :0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
+:0400DF000000005AC3
+:0400E0000000005AC2
+:0400E10000000030EB
+:0400E20000000030EA
+:0400E30000000030E9
+:0400E40000000030E8
+:0400E50000000030E7
+:0400E60000000030E6
+:0400E70000000030E5
+:0400E80000000030E4
+:0400E90000000030E3
+:0400EA0000000030E2
+:0400EB0000000030E1
+:0400EC0000000030E0
+:0400ED0000000030DF
+:0400EE0000000030DE
+:0400EF0000000030DD
+:0400F00000000030DC
+:0400F10000000030DB
+:0400F20000000030DA
+:0400F30000000030D9
+:0400F40000000030D8
+:0400F50000000030D7
 :0400F60000000030D6
 :0400F70000000032D3
 :0400F80000000041C3
@@ -151,69 +151,38 @@
 :0400FE000000005CA2
 :0400FF000000005CA1
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0401400000000002B9
+:0401410000000007B3
+:0401420000000002B7
+:040143000000003088
+:040144000000003087
+:040145000000003086
+:040146000000003085
+:040147000000003084
+:040148000000003083
+:040149000000003082
+:04014A000000003081
+:04014B000000003080
+:04014C00000000307F
+:04014D00000000307E
+:04014E00000000307D
+:04014F00000000307C
+:04015000000000307B
+:04015100000000307A
+:040152000000003079
+:040153000000003078
+:040154000000003077
+:040155000000003076
+:040156000000003075
+:040157000000003272
+:040158000000004162
+:040159000000004260
+:04015A00000000435E
+:04015B00000000445C
+:04015C000000005946
+:04015D000000005B43
+:04015E000000005C41
+:04015F000000005C40
+:040160000000006437
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (4 of 30)
processing file asmFiles/mult_procedure.asm (5 of 30)
processing file asmFiles/search.asm (6 of 30)
processing file asmFiles/test.beq_ntaken.asm (7 of 30)
processing file asmFiles/test.beq_taken.asm (8 of 30)
processing file asmFiles/test.bge_ntaken.asm (9 of 30)
processing file asmFiles/test.bge_taken.asm (10 of 30)
processing file asmFiles/test.bgeu_ntaken.asm (11 of 30)
processing file asmFiles/test.bgeu_taken.asm (12 of 30)
processing file asmFiles/test.blt_ntaken.asm (13 of 30)
processing file asmFiles/test.blt_taken.asm (14 of 30)
processing file asmFiles/test.bltu_ntaken.asm (15 of 30)
processing file asmFiles/test.bltu_taken.asm (16 of 30)
processing file asmFiles/test.bne_ntaken.asm (17 of 30)
processing file asmFiles/test.bne_taken.asm (18 of 30)
processing file asmFiles/test.branchjump1.asm (19 of 30)
processing file asmFiles/test.cfhazard_beq.asm (20 of 30)
processing file asmFiles/test.dhazard_add.asm (21 of 30)
processing file asmFiles/test.dhazard_lw.asm (22 of 30)
processing file asmFiles/test.halt.asm (23 of 30)
processing file asmFiles/test.jal.asm (24 of 30)
processing file asmFiles/test.jalr.asm (25 of 30)
processing file asmFiles/test.ldst1.asm (26 of 30)
processing file asmFiles/test.ldst2.asm (27 of 30)
processing file asmFiles/test.loadstore.asm (28 of 30)
processing file asmFiles/test.rtype.asm (29 of 30)
processing file asmFiles/test.shazard_lw.asm (30 of 30)
processing file asmFiles/count_days.asm (1 of 30)
processing file asmFiles/fib.asm (2 of 30)
--- memsim.hex	2024-09-25 10:09:05.860344392 -0400
+++ memcpu.hex	2024-09-25 10:09:05.829343216 -0400
@@ -14,27 +14,16 @@
 :04000D00FE0814E3F2
 :04000E00FFFFFFFFF2
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400230000000001D8
+:0400250000000001D6
+:0400270000000001D4
+:0400290000000001D2
+:04002B0000000001D0
+:04002D0000000001CE
+:04002F0000000001CC
+:0400310000000001CA
+:0400330000000001C8
+:0400350000000001C6
+:0400370000000001C4
 :0403C0000000001623
 :00000001FF
processing file asmFiles/mergesort.asm (3 of 30)
--- memsim.hex	2024-09-25 10:09:08.209433479 -0400
+++ memcpu.hex	2024-09-25 10:09:08.188432683 -0400
@@ -86,60 +86,60 @@
 :0400550000078463B9
 :04005600FE9FF06FAA
 :0400570000008067BE
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400C0000000000339
+:0400C1000000000734
+:0400C2000000000733
+:0400C3000000000732
+:0400C4000000000830
+:0400C500000000082F
+:0400C600000000082E
+:0400C700000000092C
+:0400C8000000000A2A
+:0400C9000000000F24
+:0400CA000000001022
+:0400CB000000001120
+:0400CC000000001719
+:0400CD000000001A15
+:0400CE00000000200E
+:0400CF00000000200D
+:0400D0000000002309
+:0400D1000000002407
+:0400D2000000002604
+:0400D3000000002603
+:0400D4000000002800
+:0400D5000000002FF8
+:0400D6000000005ACC
+:0400D7000000005ACB
+:0400D8000000005ACA
+:0400D9000000005AC9
+:0400DA000000005AC8
+:0400DB000000005AC7
+:0400DC000000005AC6
+:0400DD000000005AC5
 :0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
+:0400DF000000005AC3
+:0400E0000000005AC2
+:0400E10000000030EB
+:0400E20000000030EA
+:0400E30000000030E9
+:0400E40000000030E8
+:0400E50000000030E7
+:0400E60000000030E6
+:0400E70000000030E5
+:0400E80000000030E4
+:0400E90000000030E3
+:0400EA0000000030E2
+:0400EB0000000030E1
+:0400EC0000000030E0
+:0400ED0000000030DF
+:0400EE0000000030DE
+:0400EF0000000030DD
+:0400F00000000030DC
+:0400F10000000030DB
+:0400F20000000030DA
+:0400F30000000030D9
+:0400F40000000030D8
+:0400F50000000030D7
 :0400F60000000030D6
 :0400F70000000032D3
 :0400F80000000041C3
@@ -151,69 +151,38 @@
 :0400FE000000005CA2
 :0400FF000000005CA1
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0401400000000002B9
+:0401410000000007B3
+:0401420000000002B7
+:040143000000003088
+:040144000000003087
+:040145000000003086
+:040146000000003085
+:040147000000003084
+:040148000000003083
+:040149000000003082
+:04014A000000003081
+:04014B000000003080
+:04014C00000000307F
+:04014D00000000307E
+:04014E00000000307D
+:04014F00000000307C
+:04015000000000307B
+:04015100000000307A
+:040152000000003079
+:040153000000003078
+:040154000000003077
+:040155000000003076
+:040156000000003075
+:040157000000003272
+:040158000000004162
+:040159000000004260
+:04015A00000000435E
+:04015B00000000445C
+:04015C000000005946
+:04015D000000005B43
+:04015E000000005C41
+:04015F000000005C40
+:040160000000006437
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (4 of 30)
processing file asmFiles/mult_procedure.asm (5 of 30)
processing file asmFiles/search.asm (6 of 30)
processing file asmFiles/test.beq_ntaken.asm (7 of 30)
processing file asmFiles/test.beq_taken.asm (8 of 30)
processing file asmFiles/test.bge_ntaken.asm (9 of 30)
processing file asmFiles/test.bge_taken.asm (10 of 30)
processing file asmFiles/test.bgeu_ntaken.asm (11 of 30)
processing file asmFiles/test.bgeu_taken.asm (12 of 30)
processing file asmFiles/test.blt_ntaken.asm (13 of 30)
processing file asmFiles/test.blt_taken.asm (14 of 30)
processing file asmFiles/test.bltu_ntaken.asm (15 of 30)
processing file asmFiles/test.bltu_taken.asm (16 of 30)
processing file asmFiles/test.bne_ntaken.asm (17 of 30)
processing file asmFiles/test.bne_taken.asm (18 of 30)
processing file asmFiles/test.branchjump1.asm (19 of 30)
processing file asmFiles/test.cfhazard_beq.asm (20 of 30)
processing file asmFiles/test.dhazard_add.asm (21 of 30)
processing file asmFiles/test.dhazard_lw.asm (22 of 30)
processing file asmFiles/test.halt.asm (23 of 30)
processing file asmFiles/test.jal.asm (24 of 30)
processing file asmFiles/test.jalr.asm (25 of 30)
processing file asmFiles/test.ldst1.asm (26 of 30)
processing file asmFiles/test.ldst2.asm (27 of 30)
processing file asmFiles/test.loadstore.asm (28 of 30)
processing file asmFiles/test.rtype.asm (29 of 30)
processing file asmFiles/test.shazard_lw.asm (30 of 30)
processing file asmFiles/count_days.asm (1 of 30)
processing file asmFiles/fib.asm (2 of 30)
--- memsim.hex	2024-09-25 10:10:38.179846992 -0400
+++ memcpu.hex	2024-09-25 10:10:38.161846309 -0400
@@ -14,27 +14,16 @@
 :04000D00FE0814E3F2
 :04000E00FFFFFFFFF2
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400230000000001D8
+:0400250000000001D6
+:0400270000000001D4
+:0400290000000001D2
+:04002B0000000001D0
+:04002D0000000001CE
+:04002F0000000001CC
+:0400310000000001CA
+:0400330000000001C8
+:0400350000000001C6
+:0400370000000001C4
 :0403C0000000001623
 :00000001FF
processing file asmFiles/mergesort.asm (3 of 30)
--- memsim.hex	2024-09-25 10:10:40.495934905 -0400
+++ memcpu.hex	2024-09-25 10:10:40.475934146 -0400
@@ -86,60 +86,60 @@
 :0400550000078463B9
 :04005600FE9FF06FAA
 :0400570000008067BE
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400C0000000000339
+:0400C1000000000734
+:0400C2000000000733
+:0400C3000000000732
+:0400C4000000000830
+:0400C500000000082F
+:0400C600000000082E
+:0400C700000000092C
+:0400C8000000000A2A
+:0400C9000000000F24
+:0400CA000000001022
+:0400CB000000001120
+:0400CC000000001719
+:0400CD000000001A15
+:0400CE00000000200E
+:0400CF00000000200D
+:0400D0000000002309
+:0400D1000000002407
+:0400D2000000002604
+:0400D3000000002603
+:0400D4000000002800
+:0400D5000000002FF8
+:0400D6000000005ACC
+:0400D7000000005ACB
+:0400D8000000005ACA
+:0400D9000000005AC9
+:0400DA000000005AC8
+:0400DB000000005AC7
+:0400DC000000005AC6
+:0400DD000000005AC5
 :0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
+:0400DF000000005AC3
+:0400E0000000005AC2
+:0400E10000000030EB
+:0400E20000000030EA
+:0400E30000000030E9
+:0400E40000000030E8
+:0400E50000000030E7
+:0400E60000000030E6
+:0400E70000000030E5
+:0400E80000000030E4
+:0400E90000000030E3
+:0400EA0000000030E2
+:0400EB0000000030E1
+:0400EC0000000030E0
+:0400ED0000000030DF
+:0400EE0000000030DE
+:0400EF0000000030DD
+:0400F00000000030DC
+:0400F10000000030DB
+:0400F20000000030DA
+:0400F30000000030D9
+:0400F40000000030D8
+:0400F50000000030D7
 :0400F60000000030D6
 :0400F70000000032D3
 :0400F80000000041C3
@@ -151,69 +151,38 @@
 :0400FE000000005CA2
 :0400FF000000005CA1
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0401400000000002B9
+:0401410000000007B3
+:0401420000000002B7
+:040143000000003088
+:040144000000003087
+:040145000000003086
+:040146000000003085
+:040147000000003084
+:040148000000003083
+:040149000000003082
+:04014A000000003081
+:04014B000000003080
+:04014C00000000307F
+:04014D00000000307E
+:04014E00000000307D
+:04014F00000000307C
+:04015000000000307B
+:04015100000000307A
+:040152000000003079
+:040153000000003078
+:040154000000003077
+:040155000000003076
+:040156000000003075
+:040157000000003272
+:040158000000004162
+:040159000000004260
+:04015A00000000435E
+:04015B00000000445C
+:04015C000000005946
+:04015D000000005B43
+:04015E000000005C41
+:04015F000000005C40
+:040160000000006437
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (4 of 30)
processing file asmFiles/mult_procedure.asm (5 of 30)
processing file asmFiles/search.asm (6 of 30)
processing file asmFiles/test.beq_ntaken.asm (7 of 30)
processing file asmFiles/count_days.asm (1 of 30)
processing file asmFiles/fib.asm (2 of 30)
--- memsim.hex	2024-09-25 10:11:49.570556788 -0400
+++ memcpu.hex	2024-09-25 10:11:49.552556105 -0400
@@ -14,27 +14,16 @@
 :04000D00FE0814E3F2
 :04000E00FFFFFFFFF2
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400230000000001D8
+:0400250000000001D6
+:0400270000000001D4
+:0400290000000001D2
+:04002B0000000001D0
+:04002D0000000001CE
+:04002F0000000001CC
+:0400310000000001CA
+:0400330000000001C8
+:0400350000000001C6
+:0400370000000001C4
 :0403C0000000001623
 :00000001FF
processing file asmFiles/mergesort.asm (3 of 30)
--- memsim.hex	2024-09-25 10:11:51.955647319 -0400
+++ memcpu.hex	2024-09-25 10:11:51.916645839 -0400
@@ -86,60 +86,60 @@
 :0400550000078463B9
 :04005600FE9FF06FAA
 :0400570000008067BE
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400C0000000000339
+:0400C1000000000734
+:0400C2000000000733
+:0400C3000000000732
+:0400C4000000000830
+:0400C500000000082F
+:0400C600000000082E
+:0400C700000000092C
+:0400C8000000000A2A
+:0400C9000000000F24
+:0400CA000000001022
+:0400CB000000001120
+:0400CC000000001719
+:0400CD000000001A15
+:0400CE00000000200E
+:0400CF00000000200D
+:0400D0000000002309
+:0400D1000000002407
+:0400D2000000002604
+:0400D3000000002603
+:0400D4000000002800
+:0400D5000000002FF8
+:0400D6000000005ACC
+:0400D7000000005ACB
+:0400D8000000005ACA
+:0400D9000000005AC9
+:0400DA000000005AC8
+:0400DB000000005AC7
+:0400DC000000005AC6
+:0400DD000000005AC5
 :0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
+:0400DF000000005AC3
+:0400E0000000005AC2
+:0400E10000000030EB
+:0400E20000000030EA
+:0400E30000000030E9
+:0400E40000000030E8
+:0400E50000000030E7
+:0400E60000000030E6
+:0400E70000000030E5
+:0400E80000000030E4
+:0400E90000000030E3
+:0400EA0000000030E2
+:0400EB0000000030E1
+:0400EC0000000030E0
+:0400ED0000000030DF
+:0400EE0000000030DE
+:0400EF0000000030DD
+:0400F00000000030DC
+:0400F10000000030DB
+:0400F20000000030DA
+:0400F30000000030D9
+:0400F40000000030D8
+:0400F50000000030D7
 :0400F60000000030D6
 :0400F70000000032D3
 :0400F80000000041C3
@@ -151,69 +151,38 @@
 :0400FE000000005CA2
 :0400FF000000005CA1
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0401400000000002B9
+:0401410000000007B3
+:0401420000000002B7
+:040143000000003088
+:040144000000003087
+:040145000000003086
+:040146000000003085
+:040147000000003084
+:040148000000003083
+:040149000000003082
+:04014A000000003081
+:04014B000000003080
+:04014C00000000307F
+:04014D00000000307E
+:04014E00000000307D
+:04014F00000000307C
+:04015000000000307B
+:04015100000000307A
+:040152000000003079
+:040153000000003078
+:040154000000003077
+:040155000000003076
+:040156000000003075
+:040157000000003272
+:040158000000004162
+:040159000000004260
+:04015A00000000435E
+:04015B00000000445C
+:04015C000000005946
+:04015D000000005B43
+:04015E000000005C41
+:04015F000000005C40
+:040160000000006437
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (4 of 30)
processing file asmFiles/mult_procedure.asm (5 of 30)
processing file asmFiles/search.asm (6 of 30)
processing file asmFiles/test.beq_ntaken.asm (7 of 30)
processing file asmFiles/test.beq_taken.asm (8 of 30)
processing file asmFiles/test.bge_ntaken.asm (9 of 30)
processing file asmFiles/test.bge_taken.asm (10 of 30)
processing file asmFiles/count_days.asm (1 of 30)
processing file asmFiles/fib.asm (2 of 30)
--- memsim.hex	2024-09-25 10:13:04.919416824 -0400
+++ memcpu.hex	2024-09-25 10:13:04.895415913 -0400
@@ -14,27 +14,16 @@
 :04000D00FE0814E3F2
 :04000E00FFFFFFFFF2
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400230000000001D8
+:0400250000000001D6
+:0400270000000001D4
+:0400290000000001D2
+:04002B0000000001D0
+:04002D0000000001CE
+:04002F0000000001CC
+:0400310000000001CA
+:0400330000000001C8
+:0400350000000001C6
+:0400370000000001C4
 :0403C0000000001623
 :00000001FF
processing file asmFiles/mergesort.asm (3 of 30)
--- memsim.hex	2024-09-25 10:13:07.276506292 -0400
+++ memcpu.hex	2024-09-25 10:13:07.256505533 -0400
@@ -86,60 +86,60 @@
 :0400550000078463B9
 :04005600FE9FF06FAA
 :0400570000008067BE
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400C0000000000339
+:0400C1000000000734
+:0400C2000000000733
+:0400C3000000000732
+:0400C4000000000830
+:0400C500000000082F
+:0400C600000000082E
+:0400C700000000092C
+:0400C8000000000A2A
+:0400C9000000000F24
+:0400CA000000001022
+:0400CB000000001120
+:0400CC000000001719
+:0400CD000000001A15
+:0400CE00000000200E
+:0400CF00000000200D
+:0400D0000000002309
+:0400D1000000002407
+:0400D2000000002604
+:0400D3000000002603
+:0400D4000000002800
+:0400D5000000002FF8
+:0400D6000000005ACC
+:0400D7000000005ACB
+:0400D8000000005ACA
+:0400D9000000005AC9
+:0400DA000000005AC8
+:0400DB000000005AC7
+:0400DC000000005AC6
+:0400DD000000005AC5
 :0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
+:0400DF000000005AC3
+:0400E0000000005AC2
+:0400E10000000030EB
+:0400E20000000030EA
+:0400E30000000030E9
+:0400E40000000030E8
+:0400E50000000030E7
+:0400E60000000030E6
+:0400E70000000030E5
+:0400E80000000030E4
+:0400E90000000030E3
+:0400EA0000000030E2
+:0400EB0000000030E1
+:0400EC0000000030E0
+:0400ED0000000030DF
+:0400EE0000000030DE
+:0400EF0000000030DD
+:0400F00000000030DC
+:0400F10000000030DB
+:0400F20000000030DA
+:0400F30000000030D9
+:0400F40000000030D8
+:0400F50000000030D7
 :0400F60000000030D6
 :0400F70000000032D3
 :0400F80000000041C3
@@ -151,69 +151,38 @@
 :0400FE000000005CA2
 :0400FF000000005CA1
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0401400000000002B9
+:0401410000000007B3
+:0401420000000002B7
+:040143000000003088
+:040144000000003087
+:040145000000003086
+:040146000000003085
+:040147000000003084
+:040148000000003083
+:040149000000003082
+:04014A000000003081
+:04014B000000003080
+:04014C00000000307F
+:04014D00000000307E
+:04014E00000000307D
+:04014F00000000307C
+:04015000000000307B
+:04015100000000307A
+:040152000000003079
+:040153000000003078
+:040154000000003077
+:040155000000003076
+:040156000000003075
+:040157000000003272
+:040158000000004162
+:040159000000004260
+:04015A00000000435E
+:04015B00000000445C
+:04015C000000005946
+:04015D000000005B43
+:04015E000000005C41
+:04015F000000005C40
+:040160000000006437
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (4 of 30)
processing file asmFiles/mult_procedure.asm (5 of 30)
processing file asmFiles/search.asm (6 of 30)
processing file asmFiles/test.beq_ntaken.asm (7 of 30)
processing file asmFiles/test.beq_taken.asm (8 of 30)
processing file asmFiles/test.bge_ntaken.asm (9 of 30)
processing file asmFiles/test.bge_taken.asm (10 of 30)
processing file asmFiles/test.bgeu_ntaken.asm (11 of 30)
processing file asmFiles/test.bgeu_taken.asm (12 of 30)
processing file asmFiles/test.blt_ntaken.asm (13 of 30)
processing file asmFiles/test.blt_taken.asm (14 of 30)
processing file asmFiles/test.bltu_ntaken.asm (15 of 30)
